e2e/wdio/core/tests/card.e2e-spec.ts

```diff
@@ -0,0 +1,133 @@
+import { CardPo } from '../pages/card.po';
+import {
+    isElementDisplayed,
+    getAttributeByName,
+    getText,
+    browserIsIE,
+    getCSSPropertyByName,
+    waitForElDisplayed,
+    getElementArrayLength
+} from '../../driver/wdio';
+import { checkAttributeValueTrue, checkElArrIsClickable, checkElementText, checkElementTextValue } from '../../helper/assertion-helper';
+import CardData from '../fixtures/appData/card-content';
+
+describe('Card test suite:', function() {
+    const cardPg = new CardPo();
+
+    beforeAll(() => {
+        cardPg.open();
+    });
+
+    describe('Standard card examples:', function() {
+        it('should check card header', () => {
+            checkElArrIsClickable(cardPg.cardHeader);
+            checkElementTextValue(cardPg.cardTitle, CardData.cardTitleArr);
+            expect(isElementDisplayed(cardPg.cardAvatar)).toBe(true);
+            expect(isElementDisplayed(cardPg.cardCounter)).toBe(true);
+            expect(isElementDisplayed(cardPg.cardSubtitle)).toBe(true);
+            expect(isElementDisplayed(cardPg.cardBadge)).toBe(true);
+            expect(getText(cardPg.cardSubtitle)).toBe(CardData.cardSubtitle);
+            expect(getText(cardPg.cardBadge)).toBe(CardData.badgeText);
+        });
+
+        it('should check card content', () => {
+            checkElArrIsClickable(cardPg.cardListItems);
+            checkElementText(cardPg.cardListItems);
+            expect(getAttributeByName(cardPg.cardAttr, CardData.cardTypeAttr, 2)).toBe('list');
+        });
+    });
+
+    describe('Compact examples:', function() {
+        it('should check its compact and do basic checks', () => {
+            checkAttributeValueTrue(cardPg.compactCardAttr, CardData.compactAttr);
+            checkElementText(cardPg.compactCardHeader);
+            checkElementTextValue(cardPg.compactCardListItems, CardData.cardListItemText);
+            checkElArrIsClickable(cardPg.compactCardHeader);
+            checkElArrIsClickable(cardPg.compactCardListItems);
+            if (!browserIsIE()) {
+                expect(getCSSPropertyByName(cardPg.compactCardListItems, CardData.fontSizeAttr).value)
+                    .toBe(CardData.compactFont);
+            }
+        });
```

Comment:
skip check for IE comment?

Reply:
added comment https://github.com/SAP/fundamental-ngx/issues/4310